File-Formats/Presentation/create-read-edit-powerpoint-files-in-blazor.md

```diff
@@ -209,23 +209,23 @@ Step 11: Add the following JavaScript function in the _Host.cshtml in the Pages
 
 By executing the program, you will get the **PowerPoint document** as follows.
 
-![Blazor Server-side output PowerPoint document](Workingwith_Blazor/Output.png)
+![Blazor Server output PowerPoint document](Workingwith_Blazor/Output.png)
 
-## Client-side application
+## WASM app
 
-Step 1: Create a new C# Blazor Client-Side application project. Select Blazor App from the template and click the Next button.
+Step 1: Create a new C# Blazor WASM app project. Select Blazor App from the template and click the Next button.
 
 ![Create ASP.NET Core Web application in Visual Studio for Blazor PowerPoint document](Workingwith_Blazor/Create_project.png)
 
 Step 2: Now, the project configuration window will popup. Click Create button to create a new project with the required project name.
 
 ![Create a project name for your new project](Workingwith_Blazor/Configure_project.png)
 
-Step 3: Choose Blazor WebAssembly App and click Create button to create a new Blazor Client-Side application for .NET Core 3.0.0-preview9.
+Step 3: Choose Blazor WebAssembly App and click Create button to create a new Blazor WASM app for .NET Core 3.0.0-preview9.
 
 ![Select .NET Core, ASP.NET Core 3.0 and Blazor server_side.](Workingwith_Blazor/Core_application_Client.png)
 
-Step 4: To **create a PowerPoint document in Server-side application**, install [Syncfusion.Presentation.Net.Core](https://www.nuget.org/packages/Syncfusion.Presentation.Net.Core) to the Blazor project.
+Step 4: To **create a PowerPoint document in Server app**, install [Syncfusion.Presentation.Net.Core](https://www.nuget.org/packages/Syncfusion.Presentation.Net.Core) to the Blazor project.
```

Comment:
Here, existing content are wrong, please modify like this

**create a PowerPoint document in WASM app**